plugin.cpp

```diff
@@ -151,6 +169,30 @@ static const char *default_config = QUOTE({
 			"description": "DNP3 communication debug objects",
 			"displayName": "DNP3 debug objects",
 			"order" : "10"
+		},
+		"enableTLS": {
+			"description" : "Enable TLS encryption in outstation to master communication",
+			"type" : "boolean",
+			"default" : "false",
+			"displayName" : "Enable TLS",
+			"order" : "11",
+			"group": "TLS"
+		},
+		"TLSCAcertificate": {
+			"description": "TLS CA Certificate file name (PEM)",
```

Comment:
Used for all outstations unless overridden in the outstation list.

Reply:
What if I want to use common certificate for  some and individual or no TLS for one/few others?!